test/BatchManager-test.js

```diff
@@ -2,7 +2,7 @@ import { assert } from 'chai';
 import sinon from 'sinon-sandbox';
 
 import { makeJob, COMPONENT_NAME } from './helper';
-import BatchManager from '../lib/utils/BatchManager';
+import BatchManager from '../src/utils/BatchManager';
```

Comment:
I believe we can have source maps, even when we're running tests on the babel output. I've set this up with nyc in another package.